lib/users.ts

```diff
@@ -129,6 +82,59 @@ export const getOrCreateUser = async (userToken: JWT) => {
   }
 };
 
+/**
+ * Get all Users
+ * @returns An array of all Users
+ */
+export const getUsers = async (ability: Ability) => {
+  try {
+    checkPrivileges(ability, [{ action: "view", subject: "User" }]);
+
+    const users = await prisma.user.findMany({
+      select: {
+        id: true,
+        name: true,
+        email: true,
+        privileges: {
+          select: {
+            id: true,
+            nameEn: true,
+            nameFr: true,
+            descriptionEn: true,
+            descriptionFr: true,
+          },
+        },
+      },
+      orderBy: {
+        id: "asc",
+      },
+    });
+
+    return users;
+  } catch (e) {
+    return prismaErrors(e, []);
+  }
+};
+
+/**
+ * Get a APIUser
+ * @param userId ApiUser Id
+ * @returns ApiUser Object
+ */
+export const getApiUser = async (ability: Ability, userId: string): Promise<ApiUser | null> => {
```

Comment:
This function isn't currently used in the codebase and could be removed.  There isn't any privileges that currently restrict access to the ApiUser model.  When needed in the codebase the ApiUser object is returned as part of the Template relationship if a user has the ability to view the template object (they are one of the owners of the Template)